From 4ee8a64314c9c93496d9122ad61cc4f15688342e Mon Sep 17 00:00:00 2001
From: David Sniatkiwicz <davidsn@marvell.com>
Date: Wed, 13 Sep 2017 08:27:02 +0300
Subject: [PATCH 1999/2241] fix: doc: mvebu: a80x0: fix DPR configuration

Fix DPR configuration for configuration C, D and E,
according to the new version of the configuration file
(8K DB serdes MPP options.v11.xlsx).

Change-Id: I2857c642711df2f9068f809b70becd7d7bf0ba5a
Signed-off-by: David Sniatkiwicz <davidsn@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/44307
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Omri Itach <omrii@marvell.com>
---
 .../mvebu/a7k-a8k/armada8040-db-setup.txt          | 30 +++++++++++-----------
 1 file changed, 15 insertions(+), 15 deletions(-)

diff --git a/Documentation/mvebu/a7k-a8k/armada8040-db-setup.txt b/Documentation/mvebu/a7k-a8k/armada8040-db-setup.txt
index 7db1301..c111f4d 100644
--- a/Documentation/mvebu/a7k-a8k/armada8040-db-setup.txt
+++ b/Documentation/mvebu/a7k-a8k/armada8040-db-setup.txt
@@ -47,21 +47,21 @@ For A8040 A1 revision SoC - Please use armada-8040-db configuration,
 
 DPR Setting
 ------------
-|						| 1: A		| 2: B			| 3: C				|
-|-----------------------------------------------|---------------|-----------------------|-------------------------------|
-| Jumper-module (CON44): SLM1564		| DPR all = 1-2	| DPR all = 1-2		| DPR 96-108 = 2-3		|
-| Serdes CP0 site module (CON42): SLM1521	| DPR 2-5 = 1-2	| DPR 2-5 = 2-3		| DPR 2-5 = 1-2			|
-| Serdes CP1 site module (CON41): SLM1521	| DPR 2-5 = 1-2	| DPR 2-5 = 1-2		| DPR 2-5 = 1-2			|
-| DB8040					|		|			|				|
-|						|		|			| TDM module (CON51): SLM1448	|
-
-|						| 4: D				| 5: E					|
-|-----------------------------------------------|-------------------------------|---------------------------------------|
-| Jumper-module (CON44): SLM1564		| DPR 96-108 = 2-3		| DPR 109-112 = 2-3, others = 1-2	|
-| Serdes CP0 site module (CON42): SLM1521	| DPR 2-5 = 1-2			| DPR 2-5 = 1-2				|
-| Serdes CP1 site module (CON41): SLM1521	| DPR 2-5 = 1-2			| DPR 2-5 = 1-2				|
-| DB8040					| DPR 76,87,88 = 2-3		| DPR 81 = 2-3, DPR 119,121,123-126 =1-2|
-|						| Audio module (CON51): SLM1449	|					|
+|						| 1: A		| 2: B			| 3: C						|
+|-----------------------------------------------|---------------|-----------------------|-----------------------------------------------|
+| Jumper-module (CON44): SLM1564		| DPR all = 1-2	| DPR all = 1-2		| DPR 96-101, 103-108 = 2-3, DPR 102 = 1-2	|
+| Serdes CP0 site module (CON42): SLM1521	| DPR 2-5 = 1-2	| DPR 2-5 = 2-3		| DPR 2-5 = 1-2					|
+| Serdes CP1 site module (CON41): SLM1521	| DPR 2-5 = 1-2	| DPR 2-5 = 1-2		| DPR 2-5 = 1-2					|
+| DB8040					|		|			| DPR 84 = 2-3					|
+|						|		|			| TDM module (CON51): SLM1448			|
+
+|						| 4: D						| 5: E					|
+|-----------------------------------------------|-----------------------------------------------|---------------------------------------|
+| Jumper-module (CON44): SLM1564		| DPR 96-101,103-108 = 2-3, DPR 102 = 1-2	| DPR 109-112 = 2-3, others = 1-2	|
+| Serdes CP0 site module (CON42): SLM1521	| DPR 2-5 = 1-2					| DPR 2-5 = 1-2				|
+| Serdes CP1 site module (CON41): SLM1521	| DPR 2-5 = 1-2					| DPR 2-5 = 1-2				|
+| DB8040					| DPR 76 = 1-2, DPR 87-88 = 2-3			| DPR 81 = 2-3, DPR 23-26,119,121 =1-2	|
+|						| Audio module (CON51): SLM1449			|					|
 
 The tables below summarizes the interface configuration of each setup
 
-- 
2.7.4

